apps/aecore/src/aec_consensus_hc.erl

```diff
@@ -676,12 +675,24 @@ beneficiary_(TxEnv, Trees) ->
 
 next_beneficiary() ->
     {TxEnv, Trees} = aetx_env:tx_env_and_trees_from_top(aetx_transaction),
+    ChildBlockTime = child_block_time(),
+    case aeu_time:now_in_msecs() - aetx_env:time_in_msecs(TxEnv) of
+        T when T >= ChildBlockTime ->
+            next_beneficiary(TxEnv, Trees);
+        T ->
+            lager:debug("Not time for next block yet; sleeping ~p ms", [ChildBlockTime - T + 1]),
```

Comment:
```suggestion
            lager:debug("Too early for next block; sleeping ~p ms", [ChildBlockTime - T + 1]),
```

Reply:
I won't be bothered, this is just an intermediate step in getting the "leader"-schedule in place, etc.